### DISCUSSION TITLE
[docs] How can I make Tauri to feel like normal app

### DISCUSSION THREAD

**Discussion by ilshm** [tauri-apps/tauri#8019] [Q&A]
Hello!

Maybe a very stupid question, but I will try.

Is there any clean way to make Tauri to feel like app, not browser (remove rmb interactions, ctrl+u, ctrl+r etc, disable or replace selection)

Or only manual way blocking each bind?

**Comment by FabianLars** [comment]
Currently you'll have to disable/overwrite everything manually, yes.

On Windows the webview has a nice api that disables most shortcuts: https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2settings.arebrowseracceleratorkeysenabled?view=webview2-dotnet-1.0.2045.28 but it's not exposed in wry/tauri yet so you'll have to use the `webview2-com` crate in rust yourself do that. Also, i don't know of a similar api on Linux & macOS

**Reply by ilshm** [reply]
Thanks!
